frontend/src/core/plugins/http.ts

```diff
@@ -20,16 +20,40 @@ export async function httpGetJson<T>(path: string): Promise<T> {
       });
 }
 
+export const httpPostMultipart = async <T>(
+    url: string,
+    formData: FormData,
+    headers?: Record<string, string>
+  ): Promise<T> => {
+
+    return configuredAxios.post(url,formData, {headers}).then((res) => res ? res.data : Promise.resolve()) as Promise<T>;
+
+  };
+
 export async function httpPostJson<T>(path: string, data: any): Promise<T> {
     return configuredAxios.post(path, data).then((res) => res ? res.data : Promise.resolve()) as Promise<T>;
 }
 
+  export const httpPostDeleteFile = async <T>(
+    url: string,
+    link: string,
+  ): Promise<T> => {
+
+    return configuredAxios.post(url,link).then((res) => res ? res.data : Promise.resolve()) as Promise<T>;
+
+  };
+
+
 export async function httpPutJson<T>(path: string, data: any): Promise<T> {
-    return configuredAxios.put(path, data)
-        .then((res) => res ? res.data : Promise.resolve()) as Promise<T>;
+    return configuredAxios.put(path, data).then((res) => res ? res.data : Promise.resolve()) as Promise<T>;
 }
 
 
+
+export async function httpDeleteOldFile(path: string) {
```

Comment:
Die Methode brauchst du nicht. Da kannst du einfach das andere Delete nehmen. Vielleicht nennen wir das einfach um httpDelete, weil es ja nichts mit JSON zu tun hat.

Reply:
Paths are different. One deletes the text item + the bucket file. The other only deletes the bucket file when changing the text item.